app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java

```diff
@@ -237,7 +237,7 @@ public void onClick(DialogInterface dialog, int which) {
                         progressDialog.show();
                         //use top name entered by user to get top from database
                         //topID = binding.getTopName.getText().toString();
-                        storageReference = FirebaseStorage.getInstance().getReference("images/tops/"+topID+".png");
+                        storageReference = FirebaseStorage.getInstance().getReference("images/tops/"+topID);
```

Comment:
does this still pull the png image??

Reply:
The image is already saved as a PNG, this was an error changed because it added .png to the actual name